uber/models.py

```diff
@@ -1523,16 +1544,44 @@ def weighted_hours(self):
     def total_hours(self):
         return self.weighted_hours * self.slots
 
-    @cached_property
-    def all_staffers(self):
-        return self.session.query(Attendee).order_by(Attendee.last_first).all()
```

Comment:
Changed staff_only to staffing_only.

Btw, PyCharm has a ridiculously good refactor+rename function that seems to play well even with dynamic naming of stuff.  I have been using this a lot and even after trying to find places it missed, it seems to get it right the vast majority of the time.